### PR TITLE
Fixes issue 199

### LINKED ISSUE #199: [Feature Request] Replies Validation Rules - GET /comments/{commentId}/replies/{replyId}/votes: Create validation rules for reply's request header and params using @hapi/joi

### PR DESCRIPTION
**Pull request by tonymj76**
**What does this PR do?**
"RepliesValidationRules - GET /comments/{commentId}/replies/{replyId}/votes:
Create validation rules for reply's request header and params using @hapi/joi"

Fixes #199 

Summarise the main tasks handled in the pull request

**description of Task to be completed?**

Describe (in detail) what the task completed in the pull request does as per the relevant task

**How should this be manually tested?**

Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)

**Any background context you want to provide?**

Any pertinent information that should be considered

**What is the link to the issue on Github?**

[Issue #14](https://github.com/microapidev/comment-microapi/issues/14)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -49,6 +49,7 @@ typings/
 
 # Optional npm cache directory
 .npm
+yarn.lock
```
Comment (simply-alliv) [inline .gitignore:52]: Please remove this. You shouldn't be editing the .gitignore file.

File: utils/validationRules/Replies/getSingleReplyVoteSchema.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const Joi = require("@hapi/joi");
+/**
+ * Schema RepliesValidationRules - GET /comments/{commentId}/replies/{replyId}/votes:
+ */
+
+const getSingleReplyVoteSchema = {
```
Comment (simply-alliv) [inline utils/validationRules/Replies/getSingleReplyVoteSchema.js:6]: Your name is not a problem but just to align it with the other names please update it to this
```suggestion
const getReplyVotesSchema = {
```

File: utils/validationRules/Replies/getSingleReplyVoteSchema.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  options: {
+    allowUnknow: true,
```
Comment (simply-alliv) [inline utils/validationRules/Replies/getSingleReplyVoteSchema.js:8]: There's a typo here.

**Comment by simply-alliv** [comment]
The endpoint to get votes has an optional query parameter to filter by vote type. Please include that in the validation schema for the endpoint.

If you want an example of how to add the query parameter, view the getCommentVotesSchema.js file in the validation rules for comments.

File: .gitignore (hunk 1 above)
Comment (tonymj76) [reply .gitignore:52]: if i leave yarn.lock it will cause issue for those who are using npm not yarn

**Pull request closed without merging**